lib/checks/canFindRoots.js

```diff
@@ -0,0 +1,33 @@
+const Node = require('../node');
+const resolvesToConstant = require('./resolvesToConstant.js');
+
+function canFindRoots(equation) {
+
+  // Equation should be in the form of factor * factor = 0, factor^power = 0, or identifier = 0
+  // e.g (x - 2)^2 = 0, x(x + 2)(x - 2) = 0, x = 0
```

Comment:
does it return true for `identifier = 0` though? It seems like you only let * and ^ operator nodes through